x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -240,8 +246,8 @@ export const AllCasesList = React.memo<AllCasesListProps>(
           filterOptions={filterOptions}
           goToCreateCase={onRowClick}
           handleIsLoading={handleIsLoading}
-          isCasesLoading={isCasesLoading}
-          isCommentUpdating={isCasesLoading}
+          isCasesLoading={isLoadingCases && data.cases.length === 0}
```

Comment:
What will happen if the are no cases in the response?

Reply:
This actually a mistake. I removed the length check.